exchanges/orders.py

```diff
@@ -0,0 +1,146 @@
+from .assets import Asset
+
+def basic_order_check(action, asset_pair):
+    if action not in ['buy', 'sell']:
+        raise ValueError("Action must be 'buy' or 'sell'")
+
+    asset0, asset1 = asset_pair
+    if not isinstance(asset0, Asset) or not isinstance(asset1, Asset):
+        raise ValueError("Not recognized asset pair")
```

Comment:
Also, this deals purely with `Asset`. We could add a convenience function in `exchanges.assets` which you would call here. `validate_asset_pair` for example.